### ISSUE TITLE
Generate client for V1

### ISSUE THREAD

**Issue by parthea** [googleapis/python-error-reporting#132]
The [Error Reporting API is GA](https://cloud.google.com/error-reporting/docs/release-notes#March_31_2021) however this repo only has the generated client for [v1beta1](https://github.com/googleapis/python-error-reporting/tree/master/google/cloud/errorreporting_v1beta1).

**Comment by parthea** [comment]
Googlers see b/170493487

**Comment by meredithslota** [comment]
b/190226457 is still open. b/170493487 was closed as a "Won't Fix". Is there a path here towards getting this library to be fully autogenerated?

**Comment by daniel-sanche** [comment]
Yeah, publishing a GA release ended up slipping from our work in 2021, and then got deprioritized in the reorg.

I believe most of the hand-written content on this library is related to integrating with logging, so there may be a path to move that logic into the logging library, and changing this to fully auto-generated. Or we may be able to just remove the hand-written portions entirely. I'd have to spend some time looking closer at the library to say for sure
